testing/src/main/java/org/conscrypt/java/security/StandardNames.java

```diff
@@ -320,7 +320,7 @@ private static void addOpenSsl(String cipherSuite) {
             "TLS_PSK_WITH_AES_256_CBC_SHA");
 
     // Should be updated to match BoringSSL's defaults when they change.
-    // https://boringssl.googlesource.com/boringssl/+/master/ssl/t1_lib.c#306
+    // https://boringssl.googlesource.com/boringssl/+/master/ssl/t1_lib.cc#306
```

Comment:
Can you update this to line 289 as well?

Reply:
Yeah, sure, I just didn't know which line this was supposed to be